tests/consensus_tests/test_concurrent_shard_recovery.py

```
@@ -0,0 +1,96 @@
+    cargo build --features staging
+    pytest tests/consensus_tests/test_concurrent_shard_recovery.py -s -v
+"""
+
+import concurrent.futures
+import pathlib
+
+import requests
+
+from .assertions import assert_http_ok
+from .fixtures import create_collection, upsert_random_points
+from .utils import *
```

Ruff (0.15.21)

[error] `from .utils import *` used; unable to detect undefined names

(F403)